com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LightingShaderGraphGUI.cs

```diff
@@ -23,6 +23,11 @@ class LightingShaderGraphGUI : HDShaderGUI
             new AdvancedOptionsUIBlock(MaterialUIBlock.Expandable.Advance, ~AdvancedOptionsUIBlock.Features.SpecularOcclusion)
         };
 
+        /// <summary>
+        /// Implement your custom GUI in this function.false You'll probably want to use the MaterialUIBlock to display a UI similar to HDRP shaders.
```

Comment:
Implement your custom GUI in this function. To display a UI similar to HDRP shaders, use a MaterialUIBlock.